.git-blame-ignore-revs

```diff
@@ -1,2 +1,3 @@
 8d2463fa21386d6c0c90b2010aaee5550b505c87
-ae93e49470433648b144a64514eef708cce15143
\ No newline at end of file
+ae93e49470433648b144a64514eef708cce15143
+45739f4b57b7b824473b612aefebec8b7c2e31fd
```

Comment:
nitpick: Could possibly squeeze in the reason for the exclusion of a revision in the commit message. In this case, [sorting/formatting](https://github.com/firecracker-microvm/firecracker/commit/45739f4b57b7b824473b612aefebec8b7c2e31fd).